.chloggen/node-import.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: auto-instrumentation
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add Node.js support for `--import` flag
+
+# One or more tracking issues related to the change
+issues: [3414]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  The new UseImport configuration overrides the default injected --require flag with an --import flag that supports ESM.
+  Requires Node.js 18 or later.
```

Comment:
The important Node.js feature here is whether `module.register()` is supported and that version range is `^18.19.0 || ^20.6.0 || >=22`. It would be good to be specific here, so some user of an earlier minor version of v18 or v20 doesn't expect this to work.